db/migrate/20210524222838_create_posts.rb

```diff
@@ -0,0 +1,10 @@
+class CreatePosts < ActiveRecord::Migration[6.1]
+  def change
+    create_table :posts do |t|
+      t.string :title, null: false
```

Comment:
Eventually, if there was an admin or someway to search by title, then we might want this to have `index: true`